app/src/main/java/com/example/benchproject/app/movie/details/viewModel/MovieDetailsViewModel.kt

```diff
@@ -0,0 +1,48 @@
+package com.example.benchproject.app.movie.details.viewModel
+
+import android.util.Log
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.benchproject.app.movie.details.entity.Genres
+import com.example.benchproject.app.movie.details.entity.MovieDetails
+import com.example.benchproject.domain.movie.details.usecase.GetMovieDetailsUseCase
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+@HiltViewModel
+class MovieDetailsViewModel @Inject constructor(
```

Comment:
Unit tests

Reply:
I waiting for changes from this PR #26 to make test for view model